### ISSUE TITLE
Missing bug fixes from lantz 0.3

### ISSUE THREAD

**Issue by hgrecco** [lantzproject/lantz-core#1]
- [ ] feats.py: vascotenner/lantz@6f4c068 
   - [ ] Add documentation to limits parameter: The documentation is already there albeit much smaller
   - [x] Add stringparser in `install_requires`
- [ ] lantz-monitor.py vascotenner/lantz@e466886 : bug fixes (but see below)
- [x] action.py: vascotenner/lantz@4be7216 (adds a `log_output` argument, but see below)
- [ ] processors.py & test_feat.py: vascotenner/lantz@2a42718
   - [x] warn -> warning
   - [ ] fix test.

**Comment by hgrecco** [comment]
Regarding **lantz-monitor**. I am not sure if we should keep this within lantz. I have been using `cutelog` together with `log_to_socket` and is great. `lantz-monitor` does more than that because it understand lantz log messages. But my recomendation would be that we move this out so it can evolve independently

Regarding **log_output**. I am not sure what is the use of this argument. It toggles between showing the type or the value of the argument. We could just log both.

**Comment by vascotenner** [comment]
Regarding *log_output*. I am reading out a camera with 12mp frames at 1Gbps. I never want to log this very large datastream.

**Comment by hgrecco** [comment]
@vascotenner Makes sense but maybe we should add it as an standard option for all feats/actions 

Specific proposal:
1. Provide a way to log the type (instead of the value) for all input/ouput arguments.
2. Can be toggled on definition or during execution for each feat/dictfeat/action independently
3. Default is to log values

To resolve:
1. Should be provide different flags for inputs and one for outputs?
2. API?

**Comment by vascotenner** [comment]
Specific proposal:
2. Is there an easy way? In the old feats many of its settings are hidden in very highly complicated (and undocumented) `dict[MISSING][MISSING]` structures.

To resolve:
1. Only one flag to log all input and output values
2. API Proposal:
```python
Feat(log_values=True)
"""
:param log_values: True: Include values of input and output parameters, False: Only include type in log.
"""
```

**Comment by hgrecco** [comment]
I think we can do this in pimpmyclass
here:
https://github.com/hgrecco/pimpmyclass/blob/master/pimpmyclass/props.py#L260
and here:
https://github.com/hgrecco/pimpmyclass/blob/master/pimpmyclass/methods.py#L95

with a custom config flag.

**Comment by hgrecco** [comment]
I have pushed an initial implementation to pimpmyclass [here](https://github.com/hgrecco/pimpmyclass/commit/147297697e3559f172ccd32c25f86be1ae685307)

If will test it more and if I am happy I will make a new release and use it in lantz.
